examples/create-default-webpack-mf/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "module-federation",
+  "private": true,
+  "workspaces": {
+    "packages": [
+      "app1",
+      "app2"
+    ]
+  },
+  "scripts": {
+    "start": "lerna run --parallel start",
+    "build": "lerna run build"
+  },
+  "devDependencies": {
+    "lerna": "^4.0.0"
+  },
+  "dependencies": {
+    "zephyr-edge-contract": "^0.0.2",
```

Comment:
`zephyr edge contract` is a dependency of `zephyr webpack plugin`, you can remove the edge contract.